xarray/core/common.py

```diff
@@ -509,23 +510,26 @@ def resample(self, freq, dim, how='mean', skipna=None, closed=None,
             a function that can be called like ``how(values, axis)`` to reduce
             ndarray values along the given axis. Valid choices that can be
             provided as a string include all the usual Dataset/DataArray
-            aggregations (``all``, ``any``, ``argmax``, ``argmin``, ``max``,
-            ``mean``, ``median``, ``min``, ``prod``, ``sum``, ``std`` and
-            ``var``), as well as ``first`` and ``last``.
+            aggregations (``all``, ``any``, ``argmax``, ``argmin``,
+            ``count,`` ``max``, ``mean``, ``median``, ``min``, ``prod``,
+            ``sum``, ``std`` and ``var``), as well as ``first`` and ``last``.
```

Comment:
Either indicate that `how` has been deprecated or remove it from the doc string (like pandas did).